internal/exec/terraform.go

```diff
@@ -177,9 +176,9 @@ func ExecuteTerraform(info schema.ConfigAndStacksInfo) error {
 		     - Default values in the configuration file: these have the lowest priority
 		*/
 		if cliVars, ok := info.ComponentSection[cfg.TerraformCliVarsSectionName].(map[string]any); ok && len(cliVars) > 0 {
-			u.LogDebug("\nCLI variables (will override the variables defined in the stack manifests):")
+			log.Debug("\nCLI variables (will override the variables defined in the stack manifests):")
```

Comment:
Log statements shouldn't have formatting sequences.
```suggestion
			log.Debug("CLI variables (will override the variables defined in the stack manifests):")
```